packages/statusbar/src/statusbar.ts

```diff
@@ -129,7 +147,10 @@ export class StatusBar extends Widget implements IStatusBar {
 
   private _refreshItem(id: string) {
     const statusItem = this._statusItems[id];
-    if (statusItem.isActive()) {
+    if (
+      statusItem.isActive() &&
+      !(this.WindowZoomedLevel() && statusItem.priorityZoom === 0)
```

Comment:
Let's first test for the value comparison to avoid the function call if we can.

```suggestion
      !(statusItem.priorityZoom === 0 && this._isWindowZoomed())
```